neuraxle/metaopt/trial.py

```diff
@@ -113,48 +310,38 @@ def add_metric_results_validation(self, name: str, score: float, higher_score_is
 
         self.metrics_results[name]['validation_values'].append(score)
 
-    def save_model(self):
-        """
-        Save fitted model in the trial hash folder.
+    def get_validation_scores(self):
         """
-        hyperparams = self.hyperparams.to_flat_as_dict_primitive()
-        trial_hash = self._get_trial_hash(hyperparams)
-        self.pipeline.set_name(trial_hash).save(ExecutionContext(self.cache_folder), full_dump=True)
+        Return the validation scores for the main scoring metric.
 
-    def set_success(self):
-        """
-        Set trial status to success.
+        :return:
         """
-        self.status = TRIAL_STATUS.SUCCESS
+        return self.metrics_results['main']['validation_values']
 
-    def set_hyperparams(self, hyperparams: HyperparameterSamples):
+    def get_validation_score(self):
         """
-        Set trial hyperparams.
+        Return the latest validation score for the main scoring metric.
 
-        :param hyperparams: trial hyperparams
         :return:
         """
-        self.hyperparams = hyperparams
+        return self.metrics_results['main']['validation_values'][-1]
 
-    def set_failed(self, error: Exception):
+    def get_higher_score_is_better(self):
```

Comment:
is_higher_score_better

I'd also rename this variable everywhere in the project as "is" is a prefix.

Reply:
wrong ! it really gets if an higher score would be better for the main scoring metric !